docs/extending-minerva.rst

```diff
@@ -0,0 +1,68 @@
+Extending Minerva
+=================
+
+Creating a Minerva Plugin
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Minerva utilizes the Girder plugin system, so it will be worthwhile to familiarize yourself with their section on `Plugin Development <http://girder.readthedocs.org/en/latest/plugin-development.html>`_.
+
+For Minerva specific plugins, specifying "minerva" as a dependency within your plugin is required to ensure things load in the proper order. Take this example plugin configuration:
+
+.. code-block:: python
+
+   {
+       "name": "My Minerva Plugin",
+       "dependencies": ["minerva"]
+   }
+
+You should have a Minerva plugin that does nothing, and `can be enabled through the administration console <http://girder.readthedocs.org/en/latest/installation.html#initial-setup>`_. 
+
+
+Extending the look and feel of Minerva
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+While Minerva plugins follow the guidelines provided by Girder's client side, there are additional concerns when writing for Minerva.
```

Comment:
Or we can go with "Minerva plugins" as here, we just need to define it more explicitly above.